### PR TITLE
Add multipart support

### PR DESCRIPTION
**Pull request by ianwremmel**

### CODE REVIEW

File: packages/ciscospark/src/plugins/messages.js [hunk 1]
```diff
@@ -5,6 +5,7 @@
  */
 
 import {SparkPlugin, Page} from '@ciscospark/spark-core';
+import {isArray} from 'lodash';
```
Comment (aimeex3) [inline packages/ciscospark/src/plugins/messages.js]: just curious, in general, is it better to use lodash's isArray, or native Array.isArray?

File: packages/http-core/test/integration/spec/request.js [hunk 2]
```diff
@@ -137,7 +137,7 @@ describe(`http-core`, function() {
       });
 
       // this test fails in Safari 8+
-      it.skip(`passes cookies to endpoints on other origins`, () => {
+      nodeOnly(it)(`passes cookies to endpoints on other origins`, () => {
```
Comment (aimeex3) [inline packages/http-core/test/integration/spec/request.js]: what's the difference from nodeOnly(it) vs nodeOnly (line 116)

File: packages/ciscospark/src/plugins/messages.js (hunk 1 above)
Comment (ianwremmel) [inline packages/ciscospark/src/plugins/messages.js]: i'm pretty sure lodash is the way to go. last time i looked into it, it seemed like there were a lot of edge cases and broken implementations for Array.isArray

File: packages/http-core/test/integration/spec/request.js (hunk 2 above)
Comment (ianwremmel) [inline packages/http-core/test/integration/spec/request.js]: nodeOnly(it) is valid, nodeOnly is not. I'll fix that :)

**Comment by ianwremmel** [comment]
Fails in IE with

```
Could not complete the operation due to error 800a138f.
POST https://api.ciscospark.com/v1/messages
WEBEX_TRACKING_ID: spark-js-sdk_910c3c16-99b7-4f04-beb3-ade71a912f54_52
```
